light blue color as background for selected cells

BlueCocoa opened the pull request:
Using light blue color as background for selected cells to make it easier to read those texts labels.

```
taskFileSizeLabel
taskSpeedLabel
taskRemainingTimeLabel
taskProgressLabel
taskStatusLabel
```

BlueCocoa committed:
Using a light blue color as background for selected cells to make it easier to read those texts on labels.

BlueCocoa committed:
Assgin NSColor.clear.cgColor to background for non-selected cells.

Background should be clear for using alternating rows.

ShinCurry commented:
背景色稍微深一点
```
let selectedColor = NSColor(calibratedRed: 211.0/255.0, green: 231.0/255.0, blue: 250.0/255.0, alpha: 1.0).cgColor
```
另外修复一下 Alternating rows 问题就可以 merge 了

BlueCocoa committed:
Update selectedColor.

NSColor(calibratedRed: 211.0/255.0, green: 231.0/255.0, blue: 250.0/255.0, alpha: 1.0)

ShinCurry commented:
good job.
